### DISCUSSION TITLE
Change how exposure test slices

### DISCUSSION THREAD

**Discussion by IafSemco** [sn4k3/UVtools#759] [Ideas]
I have a saturn 2 and perhaps this is just the way it has to work. As it is now, during the exposure test, each test item is exposed separately increasing the time per layer. So for example if I did a test with 10 test objects with 1 second difference between them currently it would be 1s +2s +3s and so on, totaling 55 seconds per layer. However, if you expose them all for 1 second, then one less for 2 seconds, then one less for 3 seconds and so forth you can run that layer for 10 seconds with the same result, taking significant time off the test. Is it possible or does the firmware not allow it? (or have I not found the setting that allows it and I'm missing something?)

**Comment by sn4k3** [comment]
I have explained it many times why that won't work, but can't find the posts.
In short it invalidates the test, resin must cure all at once without pauses. The objective of this test is to get an accurate result, not cheat and have fast tests that will produce a different outcome when printing sequential cure times.

**Reply by IafSemco** [reply]
I guess I don't understand how there are pauses. Comparing to an exposure test on photo paper, if you shine a steady light on it with something covering the paper, then reveal the paper an inch at a time every second, no section is paused other than necessary between layers. What I could believe is bleed over between items but I think that is a risk in either method. anyway I appreciate the time, love the software.

**Reply by sn4k3** [reply]
There's an obligatory lift that you can't remove because firmware doesn't allow it (Only GCode printers can do that)
Lift + retract + image change takes time but also by lift will squeeze a fresh layer of resin on bottom of the test, then curing that new resin for shorter time will not be enough to solidify it to the point of creating a bad layer with partial cure (You are not only re-cure last layer but also a new thin fresh coat). To sum up, lift a incomplete cure layer can lead to no stick to the model.
-> If the lift could be removed, then it will be very similar to sequential curing and what you are asking was totally ok, but with lift it will ruin it.

**Reply by IafSemco** [reply]
that was the puzzle piece I was missing. I had been changing the GCODE on my original Saturn to do the test with a pre sliced file. Didn't know this one had to lift between exposure changes. Thanks for the info.
